Packs/XSOARStorage/Integrations/XSOARStorage/XSOARStorage.py

```diff
@@ -0,0 +1,126 @@
+import json
+import traceback
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+MAX_SIZE = int(demisto.params().get('maxsize', 1024000))
+
+
+def xsoar_store_list_command(args: Dict[str, Any]) -> CommandResults:
+
+    namespace = args.get('namespace', 'default')
+
+    data = demisto.getIntegrationContext().get(namespace)
+
+    if not data:
+        if namespace == 'default':
+            return_error(f"Namespace: <default> empty!")
+        else:
+            return_error(f"Namespace: <{namespace}> not found!")
+
+    data = [key for key in demisto.getIntegrationContext().get(namespace)]
+
+    number_of_keys = len(data)
+
+    r_data = "\n".join(data)
+
+    return CommandResults(
+        readable_output=f"{number_of_keys} key(s) found: \n {r_data}",
+        outputs_prefix=f"XSOAR.Store.{namespace}",
+        outputs={"keys": data},
+        raw_response=data
+    )
+
+
+def xsoar_store_put_command(args: Dict[str, Any]) -> CommandResults:
+
+    namespace = args.get('namespace', 'default')
+
+    key = args.get('key')
+    input_data = args.get('data')
+
+    current_data = demisto.getIntegrationContext()
+
+    if (sys.getsizeof(current_data) + sys.getsizeof(input_data)) > MAX_SIZE:
```

Comment:
Lets add to the condition a check that the data size is not exceeding 1024000 (I know you mentioned it in the description but basically if the customer is configuring a bigger size we are not preventing it)